starter-code/pizza.js

```diff
@@ -1 +1,65 @@
-// Write your Pizza Builder JavaScript in this file.
+$(document).ready(function () {
+    // Write your Pizza Builder JavaScript in this file.
+
+    // Iteration 1: Add and remove toppings
```

Comment:
The following functions are pretty similar between each other, refactoring those functions would be a great idea